tests/connection/connection.cpp

```diff
@@ -519,7 +519,7 @@ TEST(Connection, removeEquivalentVariableMethods) {
     libcellml::Variable::removeEquivalence(v2, v3);
     a = m.serialise(libcellml::Format::XML);
     EXPECT_EQ(e2, a);
-    EXPECT_THROW(libcellml::Variable::removeEquivalence(v3, v4), std::out_of_range);
+    libcellml::Variable::removeEquivalence(v3, v4);
```

Comment:
If anything, we now ought to have:

```
EXPECT_FALSE(libcellml::Variable::removeEquivalence(v3, v4));
```

meaning that `removeEquivalence()` ought to return a boolean status, etc.